### ISSUE TITLE
Question about PointNet++

### ISSUE THREAD

**Issue by gen4u** [Jiyao06/GenPose#26]
Hello!
1. When extracting features from a partial point cloud using PointNet++, I noticed that the code doesn't seem to use pre-trained weights. Did you train it from scratch?

2. Also, is it correct that Figure 5 of the paper, which shows the visualization of features, was done on the output features (1024-dim) obtained from the trained PointNet++?

Thank you

**Comment by Jiyao06** [comment]
Yes, PointNet++ is trained from scratch without using pre-trained weights. Figure 5 in the paper visualizes features from the trained PointNet++.

**Comment by sseunghyuns** [comment]
So the PoinNet++ architecture is not exactly same as the code from [here](https://github.com/erikwijmans/Pointnet2_PyTorch/tree/b5ceb6d9ca0467ea34beb81023f96ee82228f626), right?
